docs/historias-usuario.md

```diff
@@ -0,0 +1,9 @@
+# Historias de Usuario
+
+* [[HU0] Manuel quiere ver películas recien estrenadas.](https://github.com/pablo1mc315/iv_pablomc/issues/3)
```

Comment:
No entiendo muy bien qué tienen de especial las películas *recién* estrenadas. No mencionéis en las historias de usuario nada que sea irrelevante a la lógica de negocio.